### DISCUSSION TITLE
How do I restore it?

### DISCUSSION THREAD

**Discussion by opqriu** [dngray/ghacks-thunderbird-user.js#57] [Q&A]
How do I restore it?
 I'm new to Thunderbird and testing various things. 
Thank you for providing the security options for Thunderbird; 
I will use them well. 

However, is there a way to revert to the original settings just in case?
 I've deleted the user.js file, But this is no change

**Comment by opqriu** [comment]
oh backup...

**Comment by HorlogeSkynet** [comment]
Hello @opqriu, and thanks for your message.

You can read [here](https://github.com/arkenfox/user.js/wiki/2.1-User.js#-usage) how FF/TB parses, saves and interprets preferences. **TL; DR** : it's not because you restore a previous version of the `user.js` previously overridden preferences will reset to their default values (as they'll be enforced in saved `prefs.js`).

Bye :wave:
